hazelcast/src/main/java/com/hazelcast/map/impl/operation/steps/IMapOpStep.java

```diff
@@ -87,6 +87,7 @@ default boolean isStoreStep() {
      * @return name of the executor to execute this map operation step
      */
     default String getExecutorName(State state) {
+        // TODO use separate pool for tstore enabled map
```

Comment:
Do you plan to use extra pool in this PR?

Reply:
yes it is in EE PR now, please see CompactorStep in there.